Index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html lang="en">
+<html>
+<head>
+  <title>Miguel Solano's Resume</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+  <header>
+    <div>
+      <nav>
+        <ul>
+          <li><a href="#nowhere" title="Lorum ipsum dolor sit amet">Lorem</a></li>
+          <li><a href="#nowhere" title="Aliquam tincidunt mauris eu risus">Aliquam</a></li>
+          <li><a href="#nowhere" title="Morbi in sem quis dui placerat ornare">Morbi</a></li>
+          <li><a href="#nowhere" title="Praesent dapibus, neque id cursus faucibus">Praesent</a></li>
+          <li><a href="#nowhere" title="Pellentesque fermentum dolor">Pellentesque</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <div>
+    <section id="home">
+      <div>
+        <img src="" alt="profile picture">
+      </div>
+      <div>
+        <h1>
+          Miguel Ezequiel Solano Abreu
+        </h1>
+      </div>
+      <div>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem sint voluptatum laboriosam esse hic obcaecati, numquam animi facere pariatur, magnam labore omnis sapiente odio sunt aut qui nisi possimus error.
+      </div>
+    </section>
+    <section id="professional">
+      <div>
+        <h3>
+          Professional
+        </h3>
+      </div>
+      <div>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem quasi dolorem, accusantium nostrum, porro corrupti illo non cupiditate! Aliquid dolorum suscipit libero, laborum repudiandae quod earum! Minus deleniti sed aliquid.
+      </div>
+      <div>
```

Comment:
Looking at this structure, could this be a list? because I noticed like you are listing similar elements. What do you think?